src/main/java/org/fundacionjala/sfdc/Library.java

```diff
@@ -13,4 +18,13 @@
     public boolean someLibraryMethod() {
         return true;
     }
+
+    public static void getValues(Map<String, String> values, SFANewModifyPage newModifyPage) {
+        Map<String, Value> valuesToFill = new HashMap<>();
+        for (String value : values.keySet()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Iterate over the "entrySet" instead of the "keySet". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2864)